service/api_logic/teams_logic.py

```diff
@@ -18,9 +16,11 @@ def get_teams(
     query = (
         session.query(TeamIndex)
          .join(League, TeamIndex.league == League.league_id)
-         .join(Country, TeamIndex.country == Country.country_id)
          .join(Sport, TeamIndex.sport_id == Sport.sport_id)
     )
+    a = query.all()
```

Comment:
wonderful variable naming (please rewrite immediately, wash your hands and go to church afterwards so that you can atone for this shit)